src/AppServices/Services/CustomerAppService.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using Application.Models;
+using AutoMapper;
+using DomainModels;
+using DomainServices.Services;
+
+namespace AppServices;
```

Comment:
namespace sempre devem estar de acordo com as pastas

```suggestion
namespace AppServices.Services;
```